device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -322,14 +302,14 @@ public void closeNow() throws IOException
      * @throws IOException If this HTTPS request fails to send.
      * @throws URISyntaxException If the returned sas uri cannot be constructed correctly
      */
-    public FileUploadSasUriResponse getFileUploadSasUri(FileUploadSasUriRequest request) throws IOException, URISyntaxException
+    public FileUploadSasUriResponse getFileUploadSasUri(FileUploadSasUriRequest request) throws IOException
```

Comment:
isn't removing the exception being thrown a breaking change?

Reply:
Yes, but this is one of the acceptable breaking changes we agreed on making with higher ups